frontend/src/utils/format.ts

```diff
@@ -0,0 +1,14 @@
+import { GroupType } from '../types/group'
+
+export function formatGroupType(type: GroupType) {
+  switch (type) {
+    case GroupType.DAHub:
+      return 'DA hub'
+    case GroupType.ReceivingGroup:
+      return 'Receiving group'
+    case GroupType.SendingGroup:
+      return 'Sending group'
+    default:
+      return type
```

Comment:
Suggestion (non-blocking): my inclination is that `switch` statements on enums should be exhaustive and throw by default to catch unintended behavior as early as possible. Open to hearing other thinking on it though!

```suggestion
      throw new Error(`Unknown GroupType: ${type}`)
```